x-pack/test/apm_api_integration/tests/transactions/latency.ts

```diff
@@ -307,42 +307,6 @@ export default function ApiTest({ getService }: FtrProviderContext) {
         });
       });
 
-      describe('when not defined environments is seleted', () => {
```

Comment:
the edge cluster does not have any services without the environment being defined.